src/plugins/intel_cpu/src/nodes/input.cpp

```diff
@@ -355,6 +355,7 @@ void Input::cloneBlobIfRequired() {
     if (weightCache) {
         MemoryPtr ptr = *weightCache->findOrCreate(blobKey(), cloneBlob);
         memoryPtr = std::const_pointer_cast<const Memory>(ptr);
+    // todo: at offline phase we always flush subnormals to zero, probably hasSubnormals can be removed?
```

Comment:
why should we have this comment here?
I think we should rather to say:
// read_model scenario with directly loaded original model still requires this subnormal normalization

Reply:
done